### DISCUSSION TITLE
dde-file-manager 二进制和 dde-file-manager.so 的关系

### DISCUSSION THREAD

**Discussion by eagleyun-lihuiyun** [linuxdeepin/developer-center#5595] [Development Questions | 开发答疑] [answered]
在阅读 dde-file-manager 的源码时，发现文件拷贝的过程，会调用使用了dfmio库中DOperator::copyFile，其中调用了glib 的 api g_file_copy。但是实际在运行的时候，其实是调用了 dde-file-manager.so 中的 dde_file_manager::DFileCopyMoveJob::run。 
 请问 dde-file-manager.so 和 dde-file-manager 的关系，dde-file-manager 实现的 CopyWorker 会在什么场景使用？

**Comment by BLumia** [comment]
阅读的源码和实际运行的版本相符吗？是当前 git master 分支最新提交编译运行的？

**Reply by eagleyun-lihuiyun** [reply]
实际运行的版本为 5.7.5，不过源码仓库没有这个 tag。

**Reply by BLumia** [reply, marked as answer]
那可以加 deb-src 源然后 `apt source dde-file-manager` 拉取你所用的版本对应的源码。
